### PR TITLE
Redis registration store cleanup

### PR DESCRIPTION
**Pull request by msangoi**
* key renaming
* iterator cleanup

### CODE REVIEW

**Comment by jvermillard** [comment]
rationale for renaming keys?

**Comment by msangoi** [comment]
Simply homogenous naming. 
Before the change, keys for registrations and observations had a different pattern (val:key or key:val).

File: leshan-server-cluster/src/main/java/org/eclipse/leshan/server/cluster/RedisRegistrationStore.java [hunk 1]
```diff
@@ -265,11 +280,7 @@ public Registration next() {
                 if (!hasNext()) {
```
Comment (sbernard31) [inline leshan-server-cluster/src/main/java/org/eclipse/leshan/server/cluster/RedisRegistrationStore.java]: This `try (Jedis j = pool.getResource()){}` seems not useful.

**Comment by msangoi** [comment]
Good catch @sbernard31. updated. thx.

**Comment by sbernard31** [comment]
Integrated in master (commit f6aa2ac32ffa5368b975025a5cc18d95a22364cc)

**Pull request closed without merging**